composer.json

```diff
@@ -24,6 +25,16 @@
     "suggest": {
         "drupal/console": "^1"
     },
+    "autoload": {
```

Comment:
@voidtek This is not correct. You copy pasted from `oe_theme`. Please see my comment. Also, the reference to the `tests` folder is not correct, **I think**. I think it might need to be `./tests/` just like the `src` one.